opencog/spacetime/octomap/OctoMapValue.cc

```diff
@@ -0,0 +1,58 @@
+#include "OctoMapValue.h"
+
+#include <opencog/spacetime/octomap/atom_types.h>
+#include <opencog/util/exceptions.h>
+#include <opencog/atoms/base/ClassServer.h>
+#include <opencog/atoms/value/ValueFactory.h>
+
+using namespace opencog;
+
+
+void OctoValue::update() const
+{
+    auto it = (_om->get_locations_of_atom(_item)).rbegin(); //We need the last.
+    _value = {it->x(), it->y(), it->z()};
+}
+
+OctoValue::OctoValue(const HandleSeq& hseq) : FloatValue(OCTO_VALUE)
```

Comment:
This is maybe nit-picking, but it would probably be better if this took two `Handle`s, instead of a `HandleSeq`. The reason for this is to minimize API complexity: If all code everywhere always uses Handles for everything .. then we're done. But if all code everywhere has to support both Handles and handleSeq and HandleSeqSeq, and HandleFooMap and ... well, that just ends up being more code, more complexity, more bugs, and everything is harder to understand.  So its best to avoid HandleSeq most of the time.  This seems like a good example where its not really needed.